src/Components/Card/Card.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Container = styled.div`
+  margin-top: 60px;
+  display: flex;
+  flex-direction: column;
```

Comment:
Due to the allotted time, I was not able to experiment with emotion's annotation but wanted to show that I could use the library.